server/handlers/child.js

```diff
@@ -20,7 +21,8 @@ module.exports = async function(request, response) {
     slug,
     config,
     COPY: {},
-    TEXT: {}
+    TEXT: {},
+    CSV: await loadCSV(config, slug)
```

Comment:
For future reference: we may want to cache this through a service, instead of loading it directly on every re-request.